woocommerce/wc-admin
# Use a WC_Install-like approach to initialize WC Admin data

Fixes #532.

This PR introduces two new classes: `WC_Admin_Install` and `WC_Admin_Reports_Sync` that handle the initial table creation and data population as well as provide a home for database upgrade routine logic (that more closely matches WC core, making merge easier).

### Detailed test instructions:

- Use WCCOM dataset for best test results
- Disable wc-admin plugin
- Delete wc-admin lookup tables
- Activate plugin
- Verify that report table generation actions are queued and being processing
- Verify that "rebuild reports" button is under Status > Tools and does function properly
